src/stateFromMarkdown.js

```diff
@@ -1,11 +1,25 @@
 /* @flow */
 
-import MarkdownParser from './MarkdownParser';
 import {stateFromElement} from 'draft-js-import-element';
+import remark from 'remark';
+import html from 'remark-html';
+import {jsdom} from 'jsdom';
```

Comment:
Seems like issue with this test is connected to #1 and i think the problem is in stateFromElement module -> given blockquote markdown example: '> test', remark translates it to '<blockquote><p>test</p></blockquote>' which is translated by stateFromElement to 'unstyled' type block with 'test' text, instead of 'blockquote' type.